### ISSUE TITLE
String double causes overlap, long does not

### ISSUE THREAD

**Issue by CWilson4960** [phetsims/gene-expression-essentials#103]
Test device: Macbook Pro **and** Acer C720 (Chadwick)

Operating System: Mac OS Sierra 10.12.6 **and** Google Chrome OS 9901.54.0

Browser: Chrome **and** Chrome

Problem description: In stringTest=double, the Positive Transcription Factor box and RNA Polymerase box both impede onto the Regulatory region label, which takes up three lines. 
This problem does not occur in stringTest=long, where the boxes shrink down quite a bit.

Steps to reproduce:
1) type in ?stringTest=double at end of link,
2) Go to mRNA page
3) Same but with stringTest=long
phetsims/QA/issues/58

Screenshots:
![gene expression essentials_gene expression essentials screenshot](https://user-images.githubusercontent.com/31217926/32343797-624c8b98-bfca-11e7-8e3f-d57d98e46d9f.png)
![12345678901234567890123456789012345678901234567890 screenshot](https://user-images.githubusercontent.com/31217926/32343796-609faf00-bfca-11e7-8bc5-01f651f4bf15.png)

URL: https://www.colorado.edu/physics/phet/dev/html/gene-expression-essentials/1.0.0-dev.13/gene-expression-essentials_en.html
Version: 1.0.0-dev.13 2017-10-25 21:56:59 UTC
Features missing: touch
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.75 Safari/537.36
Language: en-US
Window: 1280x583
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"928741cf","branch":"master"},"axon":{"sha":"80b60293","branch":"master"},"babel":{"sha":"ce29f350","branch":"master"},"brand":{"sha":"cfca902d","branch":"master"},"chipper":{"sha":"6b338220","branch":"master"},"dot":{"sha":"85ef2bbf","branch":"master"},"gene-expression-essentials":{"sha":"a8b7efc7","branch":"master"},"joist":{"sha":"eb3a1951","branch":"master"},"kite":{"sha":"2c921cba","branch":"master"},"phet-core":{"sha":"c30d15ed","branch":"master"},"phetcommon":{"sha":"b13138d2","branch":"master"},"query-string-machine":{"sha":"c74e454e","branch":"master"},"scenery":{"sha":"92a2872d","branch":"master"},"scenery-phet":{"sha":"d394b9df","branch":"master"},"sherpa":{"sha":"0d590744","branch":"master"},"sun":{"sha":"99a0abef","branch":"master"},"tandem":{"sha":"c1e98f69","branch":"master"}}

**Comment by phet-steele** [comment]
@jbphet looks like the "Regulatory Region" and "Transcribed Region" on both screens could be widened so they could fit on two lines. Maybe? The panels have barely moved higher and doesn't alarm me.

**Comment by jbphet** [comment]
This is unlikely to be a problem in the real world.  The English version won't have any issues with this, so it would only be a problem if a translator added a 2nd newline to the string.  We leave it up to the translators to verify that their translation looks reasonable.  I **could** add a max height, but I feel like that is more likely to cause other problems in the future rather than prevent issues.

I discussed this with @phet-steele, and he concurred.  Closing.
